groups/infrastructure/locals.tf

```diff
@@ -8,8 +8,7 @@ locals {
   kms_key_alias                   = local.stack_secrets["kms_key_alias"]
   vpc_name                        = local.stack_secrets["vpc_name"]
   notify_topic_slack_endpoint     = local.stack_secrets["notify_topic_slack_endpoint"]
-  ingress_cidrs_private           = concat(local.management_private_subnet_cidrs, local.application_cidrs)
-  ingress_prefix_list_ids         = [data.aws_ec2_managed_prefix_list.admin_cidr.id]
+  ingress_cidrs_private           = concat(local.application_cidrs)
```

Comment:
I'd be very careful about just removing these entirely. There are other albs set up in this stack that may very well require management/admin/private ingress